### DISCUSSION TITLE
Yetiforce Online/Offline

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#17283] [General]
What would be the approach to let Yetiforce store data temporarily offline (when latency goes bad of Internet gets cut off) and sync data with database when connection goes back on track? 
Came across that designing some onsite reporting instance for construction industry... remote area, deep basements etc..

**Comment by mariuszkrzaczkowski** [comment]
with current technology it is complicated
